plugins/woocommerce-admin/client/products/sections/pricing-section.tsx

```diff
@@ -56,6 +75,51 @@ export const PricingSection: React.FC = () => {
 		'woocommerce'
 	);
 
+	const { dateFormat, timeFormat } = useSelect( ( select ) => {
+		const { getOption } = select( OPTIONS_STORE_NAME );
+		return {
+			dateFormat: ( getOption( 'date_format' ) as string ) || 'F j, Y',
```

Comment:
Not something we need to implement in this PR, but just thinking out loud that it would be nice to have defaults like this stored somewhere.

I know we have some functions around intervals in the date package that has this for intervals, but feels too specific for this particular use case.

https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/date/src/index.ts#L795

Reply:
Agreed. Maybe it could belong in`date`.